scanapi/reporter.py

```diff
@@ -43,6 +43,24 @@ def write(self, results):
         logger.info("\nThe documentation was generated successfully.")
         logger.info(f"It is available at {abspath(self.output_path)}")
 
+    def write_without_generating_report(self, results):
+        """Part of the Reporter instance that is responsible for writing the
+        results without generating the scanapi-report.html.
+
+        Args:
+            results [generator]: generator of dicts resulting of Request run().
+
+        Returns:
+            None
+        """
+        logger.info("Writing results without generating report")
+        for r in results:
+            if logger.root.level != logging.DEBUG:
+                for test in r["tests_results"]:
+                    logger.info(f" [{test['status'].upper()}] {test['name']}")
+                    if test["status"] == "failed":
```

Comment:
We have a constant for 'failed', so I suggest to use it instead of the hard coded value. What do you think?

```suggestion
                    if test["status"] == TestStatus.FAILED:
```

Reply:
Oh I didn't realize it exists. Thanks for the suggested change :)